src/alt/store/AltStore.js

```diff
@@ -55,7 +55,7 @@ class AltStore {
           return actionHandler.call(model, payload.data, payload.action)
         }, payload)
 
-        if (result !== false && !this.preventDefault) this.emitChange()
+        if (result !== false && !this.preventDefault && this.StoreModel.config.shouldEmitChange) this.emitChange()
```

Comment:
If `shouldEmitChange` is false, how will emitChange() ever be called? It should still be called if `setState` was called at least once.

Since `shouldEmitChange` is not a function anymore, a better name might be `alwaysEmitChange`.